libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
```

Comment:
Maybe we should extend devlib's `KernelVersion` instead of having the regex and uname call in here? I don't mind though, up to you.